incubator/virtualcluster/pkg/util/mccontroller/mccontroller.go

```diff
@@ -463,6 +475,70 @@ func (c *MultiClusterController) processNextWorkItem() bool {
 	return false
 }
 
+func (c *MultiClusterController) FilterObjectFromSchedulingResult(req reconciler.Request) bool {
+	var nsName string
+	if c.objectKind == "Namespace" {
+		nsName = req.Name
+	} else {
+		nsName = req.Namespace
+	}
+
+	if filterSuperClusterRelatedObject(c, req.ClusterName, nsName) {
+		c.Queue.Done(req)
+		c.Queue.AddAfter(req, time.Minute)
```

Comment:
Is the reason of requeuing the request is to handle the case that object owner super cluster is changed? I think this can be done in the patroller (it needs to be changed anyway).